src/containers/ImageUploader/components/ImageForm.tsx

```diff
@@ -91,13 +90,10 @@ const FormWrapper = ({ inModal, children }: { inModal?: boolean; children: React
   return <Form>{children}</Form>;
 };
 
-type OnUpdateFunc = (imageMetadata: UpdatedImageMetadata, image: string | Blob, id: number) => void;
-type OnCreateFunc = (imageMetadata: NewImageMetadata, image: string | Blob) => void;
-
 interface Props {
-  image?: ImageApiType;
-  licenses: License[];
-  onUpdate: OnCreateFunc | OnUpdateFunc;
+  image?: IImageMetaInformationV2;
+  licenses: ILicense[];
+  onSubmitFunc: (imageMetadata: INewImageMetaInformationV2, image: string | Blob) => void;
```

Comment:
Selv om typen er lik hadde det ikke vært feil å splitte det opp til to forskjellige funksjoner.